slick.groupitemmetadataprovider.js

```diff
@@ -174,12 +174,6 @@
     };
   }
 
-  // exports
-  $.extend(true, window, {
-    Slick: {
-      Data: {
-        GroupItemMetadataProvider: GroupItemMetadataProvider
-      }
-    }
-  });
+  Slick.Data = Slick.Data || { };
+  Slick.Data.GroupItemMetadataProvider = GroupItemMetadataProvider;
 })(jQuery);
```

Comment:
this file only has 1 more jQuery use which can also be removed easily

```diff
    function handleGridClick(e, args) {
-      var $target = $(e.target);
+      var targetElm = e.target;
      var item = this.getDataItem(args.row);
-      if (item && item instanceof Slick.Group && $target.hasClass(options.toggleCssClass)) {
+      if (item && item instanceof Slick.Group && targetElm.classList.contains(options.toggleCssClass)) {
```

Reply:
Fixed